ergo-lib/src/wallet/box_selector/simple.rs

```diff
@@ -76,11 +76,7 @@ impl<T: ErgoBoxAssets + Clone> BoxSelector<T> for SimpleBoxSelector {
         // reverse, so they'll be sorted by descending order (boxes with target tokens will be first)
         sorted_inputs.reverse();
         for b in sorted_inputs {
-            let value_change_amt: u64 = if target_balance > selected_boxes_value {
-                0
-            } else {
-                selected_boxes_value - target_balance
-            };
+            let value_change_amt: u64 = selected_boxes_value.saturating_sub(target_balance);
```

Comment:
is it the same?

Reply:
> is it the same?

Yes. https://rust-lang.github.io/rust-clippy/master/index.html#manual_saturating_arithmetic